pkg/txnutil/gc/gc_manager.go

```diff
@@ -29,6 +28,11 @@ import (
 	"go.uber.org/zap"
 )
 
+// failedFeedDataRetentionTime is the duration during which data related to a
+// failed feed will be retained, and beyond which point the data will be deleted
+// by garbage collection.
+const failedFeedDataRetentionTime = 24 * time.Hour
```

Comment:
don't hard code it,  maybe use gc ttl is better

Reply:
rename as `gcTTL`